src/contracts/StakingChef.sol

```diff
@@ -0,0 +1,184 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/utils/math/SafeMath.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "./Chef.sol";
+
+contract StakingChef is Chef {
+    using SafeMath for uint256;
+    Staker[] private _stakes;
+    uint256 private _totalAllocPoint;
+    uint256 private _lifetimeShareValue;
+    uint256 public UserStake;
+    MetricToken public metric;
```

Comment:
instead of defining MetricToken in both the StakingChef and the <base>Chef, can we make the variable _metric in the <base>Chef accessible to subclasses?

Reply:
@gnomadic @addyxvii - yes that makes sense - I have updated my PR to reflect this